testing/trino-testing/src/main/java/io/trino/testing/AbstractTestAggregations.java

```diff
@@ -233,7 +235,17 @@ public void testCountDistinct()
     @Test
     public void testDistinctGroupBy()
     {
-        assertQuery("SELECT COUNT(DISTINCT clerk) AS count, orderdate FROM orders GROUP BY orderdate ORDER BY count, orderdate");
+        @Language("SQL") String query = "" +
```

Comment:
Can we make test future proof, so we notice when it longer provides coverage (all used aggregates become spillable)?

Reply:
> all used aggregates become spillable

I'm pretty much sure this will never happen, so I will leave it as is